src/functions/pipedrive/find-deal-by-id/index.ts

```diff
@@ -0,0 +1,16 @@
+import { handlerPath } from "@libs/handler-resolver";
+
+export default {
+  handler: `${handlerPath(__dirname)}/handler.main`,
+  events: [
+    {
+      httpApi: {
+        method: 'get',
+        path: '/getLeadOrDealById/{rowtype}/{id}',
```

Comment:
According to the updated spec this path should be getDealById/{id}. Please check the path for all functions match the spec otherwise the client will not work.